src/sage/functions/exp_integral.py

```diff
@@ -601,8 +601,8 @@ class Function_log_integral_offset(BuiltinFunction):
         1/log(x)
         sage: f.integrate(x)
         -x*log_integral(2) + x*log_integral(x) - Ei(2*log(x))
-        sage: Li(x).integrate(x,2.0,4.5)
-        -2.5*log_integral(2) + 5.799321147411334
+        sage: Li(x).integrate(x,2.0,4.5).n()
+        3.18641169711760
```

Comment:
Add an `abs tol` here as a preventative measure? The following test uses `1e-15`.

Reply:
It's actually an exact value, `-Ei(2*log(9/2)) + Ei(2*log(2)) + 9/2*log_integral(9/2) - 9/2*log_integral(2)` - as computed by maxima 5.46.  I gather that the discrepancy is due to 5.45 and 5.46 treating floating point input a bit differently.

I can cut few digits and replace them with `...`

Reply:
I am more in favor of adding a tolerance myself. Numerical results like this one should always have a tolerance. As an alternative, we could use, say, `.n(digits=10)` which is not unlike `...` but more purposeful in my opinion.

Reply:
it's not really a numerical result, in the sense it's evaluating a symbolic expression in functions Maxima knows how to evaluate well.
I'll add `digits=10`.

Reply:
fixed by [79d7c28](https://github.com/sagemath/sage/pull/35619/commits/79d7c2862d0841d50ded76a47ead039d59a321e1)